plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_core.py

```diff
@@ -59,4 +59,4 @@ def launch(
             )
             runs.append(ray_obj)
 
-    return [ray.get(run) for run in runs]
+    return ray.get(runs)
```

Comment:
This is better style for Ray. If the elements of `runs` were not already Ray ObjectRef instances, this would expression would cause each run to be blocked by the previous one.

https://docs.ray.io/en/master/auto_examples/tips-for-first-time.html

Reply:
nice! however pls still cast this to `JobReturn` :)